components/dash-core-components/src/fragments/Dropdown.react.js

```diff
@@ -158,6 +160,7 @@ const Dropdown = props => {
                 value={value}
                 onChange={onChange}
                 onInputChange={onInputChange}
+                closeMenuOnSelect={closeMenuOnSelect}
```

Comment:
This is given in the `{...pick(RDProps, props)}` so it doesn't need to be given again.